File: TradeHarborApi/Models/Notification/Notification.cs
namespace TradeHarborApi.Models.Notification
{
    /// <summary>
    /// Represents a notification entity.
    /// </summary>
    public class Notification
    {
        /// <summary>
        /// Gets or sets the message content of the notification.
        /// </summary>
        public string Message { get; set; } = string.Empty;

        /// <summary>
        /// Gets or sets the timestamp when the notification was created.
        /// </summary>
        public DateTime CreatedTimestamp { get; set; }

        /// <summary>
        /// Gets or sets the unique identifier of the notification.
        /// </summary>
        public string NotificationId { get; set; } = string.Empty;
    }
}

File: TradeHarborClient/src/app/models/api-responses/notification.ts
/**
 * Represents a notification with basic information.
 */
export class Notification {
    /**
     * The main message of the notification.
     */
    message: string = '';
  
    /**
     * The unique identifier for the notification.
     */
    notificationId: string = '';
  
    /**
     * The timestamp when the notification was created.
     */
    createdTimestamp: Date = new Date();
  }
  